docs/References/Glossary.md

```diff
@@ -0,0 +1,58 @@
+# Glossary
+
+## Capture Volume
+3-dimensional area (volume) with sufficient camera coverage to support 3D tracking.
+
+## Calibration
+
+## Charuco Board
+
+## Media pipe
+
+## Processing Stages
+
+ - **Stage 1 - Record Videos**
+   -  Record raw videos from attached USB webcams and timestamps for each frame 
+   -  Raw Videos saved to `FreeMoCap_Data/[Session Folder]/RawVideos`
+
+ - **Stage 2 - Synchronize Videos**
```

Comment:
Would just make a note that this is only for videos from cameras connected to the computer at the moment - not for videos from external sources like GoPros